Ginger/Ginger/Actions/ActionEditPages/Communication/ActeMailEditPage.xaml.cs

```diff
@@ -74,6 +81,61 @@ private void Bind()
             }
             ShowDisplayNameOption();
         }
+        private void xcbValidationRequired_checked(object sender, RoutedEventArgs e)
+        {
+            IsValidationRequired.IsChecked = true;
+            CertificateStackPanel.Visibility = Visibility.Visible;
+        }
+        private void xcbValidationRequired_unchecked(object sender, RoutedEventArgs e)
+        {
+            IsValidationRequired.IsChecked = false;
+            CertificateStackPanel.Visibility = Visibility.Collapsed;
+        }
+        private void BrowseButton_Certificate(object sender, RoutedEventArgs e)
+        {
+            System.Windows.Forms.OpenFileDialog dlg = new System.Windows.Forms.OpenFileDialog();
+
+            dlg.DefaultExt = "*.crt";
```

Comment:
will it accept '.CRT' extension?

Reply:
Will accepts all the certifcate types which are possible.